backend/src/index.ts

```diff
@@ -213,111 +357,45 @@ async function main(): Promise<void> {
   // regularly publish information about validators, proposals, staking pools, and online nodes
   const regularPublishNetworkInfo = async (): Promise<void> => {
     try {
-      const epochStats = await queryEpochStats();
-
-      const onlineNodesCount = await queryOnlineNodesCount();
-
-      const originalStakingNodes = await extendWithTelemetryInfo([
-        ...epochStats.stakingNodes.values(),
+      const epochData = await queryEpochData();
+      const telemetryInfo = await queryTelemetryInfo(
+        epochData.validators.map((validator) => validator.accountId)
+      );
+      await Promise.all([
+        Promise.race([
+          getContractStakeMap(epochData.validators, contractBalances),
+          wait(2500),
```

Comment:
This value should be constant, it's bailout time - we should give some data to the client, not wait until everything is resolved.